src/fmi4c_utils.c

```diff
@@ -7,17 +7,17 @@
 //! @brief Concatenates model name and function name into "modelName_functionName" (for FMI 1)
 //! @param modelName FMU model name
 //! @param functionName Function name
+//! @param concatBuffer A buffer in which to concatenate modelName_functionName, this buffer must be FILENAME_MAX big
 //! @returns Full (concatenated) function name
-const char* getFunctionName(const char* modelName, const char* functionName) {
-    if(modelName == NULL || modelName[0] == '\0') {
-        return functionName;    //!< Do not change function name if model name is empty
+const char* getFunctionName(const char* modelName, const char* functionName, char* concatBuffer) {
+    if(modelName == NULL || strlen(modelName) == 0) {
+        // Do not change function name if model name is empty, in this case there is no need to sue the buffer
+        return functionName;
```

Comment:
typo ("sue" should be "use") but does not really matter

Reply:
will fix